azuredevops.go

```diff
@@ -93,16 +93,21 @@ func runAzureDevOpsHandler(c *gin.Context) {
 	i := importBody{}
 	err := c.BindJSON(&i)
 	if err != nil {
-		c.Error(err)
-		c.JSON(http.StatusBadRequest, err.Error())
+		ginutil.WriteInvalidBindError(c, err,
+			"One or more parameters failed to parse when reading the request body for branch object to update.")
```

Comment:
This seems a little copy-pasted

```suggestion
			"One or more parameters failed to parse when reading the request body for import details.")
```